frontend/src/views/WheresMyGene/components/HeatMap/components/XAxisChart/index.tsx

```diff
@@ -1,75 +1,67 @@
-import { init } from "echarts";
-import { useEffect, useRef, useState } from "react";
-import { EMPTY_OBJECT } from "src/common/constants/utils";
+import { useEffect, useState } from "react";
 import { useDeleteGenesAndCellTypes } from "../../hooks/useDeleteGenesAndCellTypes";
-import { useUpdateXAxisChart } from "../../hooks/useUpdateXAxisChart";
-import { CHART_LEFT_PADDING } from "../../style";
-import { getHeatmapWidth, Y_AXIS_CHART_WIDTH_PX } from "../../utils";
-import { XAxisWrapper, XAxisContainer } from "./style";
+import { CHART_LEFT_PADDING, SELECTED_STYLE } from "../../style";
+import {
+  getHeatmapWidth,
+  X_AXIS_CHART_HEIGHT_PX,
+  Y_AXIS_CHART_WIDTH_PX,
+} from "../../utils";
+import {
+  XAxisWrapper,
+  XAxisContainer,
+  XAxisLabel,
+  GeneButtonStyle,
+} from "./style";
+import { formatLabel } from "../YAxisChart";
```

Comment:
low pri nit: ideally if we could extract `formatLabel` to a common util folder (maybe `frontend/src/views/WheresMyGene/components/HeatMap/utils.ts`?, then XAxis component wouldn't need to depend on YAxis component at all!